packages/system/src/Location/Components/LocationSearchInput.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC } from 'react';
```

Comment:
Not sure if this will be used elsewhere.. but maybe!

Reply:
I expect it will 😃